Deploy-CAM.ps1

```diff
@@ -4222,6 +4269,16 @@ else {
         radiusServerPort = $radiusServerPort 
         radiusSharedSecret = $radiusSharedSecret
     }
+
+    if ($camManagementUserGroup -eq $null -and (-not $ignorePrompts)) {
+        $camManagementUserGroup = Read-Host "Enter the User Group Name or the Distinguished Name for the User Group to log into the CAM Management Interface. Default is 'Domain Admins'. (eg, 'Domain Admins' or 'CN=Domain Admins,CN=Users,DC=example,DC=com')"
```

Comment:
I think that in the case where someone wants to customise the management user group, we should require the full DN for the group so that there is no possibility of getting the wrong group if we try to figure out the DN later (as we currently do for the "Domain Admins" group).